state-transition/core/state_processor_genesis.go

```diff
@@ -65,6 +65,7 @@ func (sp *StateProcessor[_]) InitializePreminedBeaconStateFromEth1(
 		Eth1Data: &ctypes.Eth1Data{},
 		ExecutionPayload: &ctypes.ExecutionPayload{
 			ExtraData: make([]byte, ctypes.ExtraDataSize),
+			EpVersion: constants.GenesisVersion,
```

Comment:
I believe this does not really have any practical effect, but the version was not set right